Frontend/src/pages/ProfileCompletion.jsx

```
@@ -0,0 +1,188 @@
+      <h2 className="main-title">Create Your Academic Persona</h2>
+      {step === 1 && (
+        <div className="profile-step">
+          <h3 className="step-title">Step 1: Basic Information</h3>
+          <input type="text" name="university" placeholder="University Name" value={formData.university} onChange={handleChange} className={`input-box ${errors.university ? "error" : ""}`} />
+          <input type="text" name="department" placeholder="Department" value={formData.department} onChange={handleChange} className={`input-box ${errors.department ? "error" : ""}`} />
+          <h3 className={`interest-title ${errors.interests ? "error-text" : ""}`}>Select Fields of Interest</h3>
+          <div className="interest-box">
+            {availableInterests.map((interest) => (
+              <span 
+              key={interest}
+              role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)